Assignment2.yaml

```diff
@@ -0,0 +1,415 @@
+        required: false
+        type: "string"
+      - name: "model"
+        in: "query"
+        required: false
+        type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            type: "array"
+            items:
+             $ref: "#/definitions/Vehicle"
+  /Vehicle/{VehicleId}:
+    get:
+      tags:
+      - "Vehicle"
+      summary: "Find Vehicle by ID"
+      description: "Returns a single vehicle"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "VehicleId"
+        in: "path"
+        description: "ID of Vehicle to return"
+        required: true
+        type: "number"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/Vehicle"
+  /Vehicle/schema:
+    get:
+      tags:
+      - "Vehicle"
+      summary: "Find Vehicle Schema"
+      description: "Returns a json schema"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/Vehicle"
+  /People:
+    get:
+      tags:
+      - "People"
+      summary: "Find list of People"
+      description: "get all the people resources"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "name"
+        in: "query"
+        required: false
+        type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/people"
+  /People/{PeopleId}:
+    get:
+      tags:
+      - "People"
+      summary: "Find people by ID"
+      description: "get a specific people resource"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "PeopleId"
+        in: "path"
+        description: "ID of people to return"
+        required: true
+        type: "number"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/people"
+  /People/schema:
+    get:
+      tags:
+      - "People"
+      summary: "Return Schema"
+      description: "Returns a json Schema"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/people"
+  /starship:
+    get:
+      tags:
+      - "Starship"
+      summary: "Find list of starships"
+      description: "Returns a List of starship"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "name"
+        in: "query"
+        required: false
+        type: "string"
+      - name: "model"
+        in: "query"
+        required: false
+        type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/starship"
+  /starship/{starshipId}:
```

Comment:
Should be /starships. (plural)
Check in your postman if the url is working or not.